test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -6,7 +6,7 @@
   "compilationOptions": {},
   "targets": {
     ".NETCoreApp,Version=v6.0": {
-      "Microsoft.Azure.WebJobs.Script.WebHost/4.1.0": {
+      "Microsoft.Azure.WebJobs.Script.WebHost/4.1.1": {
```

Comment:
Not sure if all these deps file changes are needed - we'll need to carefully review this.

Reply:
Yes, this jumped out to me as worrisome as well. Ideally, this file should only inject references specific to the sole dependency I've introduced, Microsoft.Security.Utilities. However, a specific unit test failed on my machine making that focused change, and so I went ahead and baselined. 

Several other tests fail as well, so my best bet is probably to consult with any machine/testing setup guidance you have to get the broader functionals working. One test appears to indicate Java be installed. Others require a missing connection string.

I'll be giving this PR attention tomorrow (to fix the API factoring improvements we discussed offline) and I'll reset this deps.json file. If you have a suggestion on ensuring all tests pass, let me know, otherwise will go digging around for info in the repo.

Reply:
I've reverted everything not relevant to MS.Security.Utilities.